pkg-manager/plugin-commands-installation/src/update/index.ts

```diff
@@ -238,9 +238,9 @@ async function interactiveUpdate (
       return this.styles.primary(this.selected.name)
     },
     styles: {
-      dark: chalk.white,
+      dark: chalk.gray,
```

Comment:
try to use `chalk.reset` instead

Reply:
Done https://github.com/pnpm/pnpm/pull/7454/commits/997502ad88af8e0eb6f9e017b31bc9df33e6237f

Reply:
How I can run the local copy of `pnpm`?

Reply:
try with the steps described in the contributing.md https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md#setting-up-the-environment

Reply:
In my terminal it looks fine. If it looks fine in your terminal, we can accept this change.

Reply:
Looks good 👍